util/crypto/src/secp/signature.rs

```diff
@@ -50,15 +50,6 @@ impl Signature {
         Signature(sig)
     }
 
-    /// Check if this is a "low" signature.
-    pub fn is_low_s(&self) -> bool {
```

Comment:
where is this function used?

Reply:
Not used in current CKB, this PR intends to remove it to avoid confusion.